tools/deviceGen/input/device_features.json

```diff
@@ -0,0 +1,2666 @@
+                "VK_EXT_external_memory_host",
+                "[TODO] need impl/more research",
+                "VK_AMD_buffer_marker"
+            ]
+        },
+        {
+            "type": "bool",
+            "name": "bufferMarkerAMD",
+            "value": false
+        },
+        {
+            "groupComment": [
+                "[DEPRECATED] Vulkan 1.2 Core",
+                "VK_KHR_shader_atomic_int64",
+                "[EXPOSE AS LIMIT]",
+                "ShaderClockFeaturesKHR",
+                "VK_KHR_shader_clock"
+            ]
+        },
+        {
+            "declare": true,
+            "type": "bool",
+            "name": "shaderDeviceClock",
+            "expose": false
+        },
+        {
+            "type": "bool",
+            "name": "shaderSubgroupClock",
+            "value": "shaderDeviceClock",
+            "expose": false
+        },
+        {
+            "groupComment": [
+                "[DO NOT EXPOSE] just reserved numbers, extension never shipped",
+                "VK_AMD_extension_183",
+                "[DO NOT EXPOSE] Too vendor specific",
+                "VK_AMD_pipeline_compiler_control",
+                "[TODO LATER] Requires changes to API",
+                "VK_EXT_calibrated_timestamps",
+                "[DEPRECATED] Superseded by VK_AMD_shader_core_properties",
+                "VK_AMD_shader_core_properties",
+                "[DO NOT EXPOSE] just reserved numbers, extension never shipped",
+                "VK_AMD_extension_187",
+                "[DO NOT EXPOSE] We don't want to be on the hook with the MPEG-LA patent trolls",
+                "VK_KHR_video_decode_h265",
+                "[TODO] this one isn't in the headers yet",
+                "GlobalPriorityQueryFeaturesKHR",
+                "VK_KHR_global_priority"
+            ]
+        },
+        {
+            "declare": true,
+            "type": "VkQueueGlobalPriorityKHR",
+            "name": "globalPriority",
+            "expose": false
+        },
+        {
+            "groupComment": [
+                "[DO NOT EXPOSE] just reserved numbers, extension never shipped",
+                "VK_AMD_extension_183",
+                "[DO NOT EXPOSE] Too vendor specific",
+                "VK_AMD_pipeline_compiler_control",
+                "[TODO LATER] Requires changes to API",
+                "VK_EXT_calibrated_timestamps",
+                "[DEPRECATED] Superseded by VK_AMD_shader_core_properties",
+                "VK_AMD_shader_core_properties",
+                "[DO NOT EXPOSE] just reserved numbers, extension never shipped",
+                "VK_AMD_extension_187",
+                "[DO NOT EXPOSE] We don't want to be on the hook with the MPEG-LA patent trolls",
+                "VK_KHR_video_decode_h265",
+                "[TODO] this one isn't in the headers yet",
+                "GlobalPriorityQueryFeaturesKHR",
+                "VK_KHR_global_priority"
+            ]
+        },
```

Comment:
this seems repeated/weirdly out of order

Reply:
missing a whole bunch of comments from previous `SPhysicalDeviceFeatures` between lines 915 and 958